swagger.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 // Name is a unique name be used to register swag instance.
-const Name = "swagger"
+const Name = "Swagger"
```

Comment:
I don't think changing this value may be a good idea.

Reply:
`const Name = "swagger"` will generate `docTemplateswagger` and `SwaggerInfoswagger` which is OK from the perspective of a linter, but a bit difficult to read from a human's perspective. That's why I capitalized it. And this change also didn't break any other part of the full repo.

After reading this comment, if you still believe that it is a better idea to keep it as `"swagger"`, kindly let me know. I will revert it back.

Thank you.

Reply:
The generates SwaggerInfo if Instance name == "swagger" and SwaggerInfo+InstanceName if not.
```
var SwaggerInfo{{ if ne .InstanceName "Swagger" }}{{ .InstanceName }} {{- end }} = &swag.Spec{
```

Please roll back that change.

Reply:
done.